docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -309,5 +309,98 @@ Both visualizations are now displayed on your custom dashboard.
 IMPORTANT: By default, Discover shows data for the last 15 minutes. If you have a time-based index
 and no data displays, you might need to increase the time range.
 
+[[open-telemetry-aws-lambda-elastic]]
+==== AWS Lambda Support
+
+AWS Lambda functions can be instrumented with OpenTelemetry and monitored with Elastic Observability.
+
+To get started, follow the official AWS Distro for OpenTelemetry Lambda https://aws-otel.github.io/docs/getting-started/lambda[getting started documentation] and configure the OpenTelemetry Collector to output traces and metrics to your Elastic cluster.
+
+[[open-telemetry-aws-lambda-elastic-java]]
+==== Instrumenting AWS Lambda Java functions
+
+NOTE: For a better startup time, we recommend using SDK-based instrumentation, i.e. manual instrumentation of the code, rather than auto instrumentation.
+
+To instrument AWS Lambda Java functions, follow the official https://aws-otel.github.io/docs/getting-started/lambda/lambda-java[AWS Distro for OpenTelemetry Lambda Support For Java].
+
+Noteworthy configuration elements:
+
+* AWS Lambda Java functions should extend `com.amazonaws.services.lambda.runtime.RequestHandler`,
++
+[source,java]
+----
+public class ExampleRequestHandler implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent event, Context context) {
+        // add your code ...
+    }
+}
+----
+
+* When using SDK-based instrumentation, frameworks you want to gain visibility of should be manually instrumented
+** The below example instruments https://square.github.io/okhttp/4.x/okhttp/okhttp3/-ok-http-client/[OkHttpClient] with the OpenTelemetry instrument https://search.maven.org/artifact/io.opentelemetry.instrumentation/opentelemetry-okhttp-3.0/1.3.1-alpha/jar[io.opentelemetry.instrumentation:opentelemetry-okhttp-3.0:1.3.1-alpha]
+  +
```

Comment:
This might've been my fault. There's a space here that shouldn't be here and it breaks the rendering.

```suggestion
+
```